### ISSUE TITLE
Create front end for project

### ISSUE THREAD

**Issue by giantflyingegg** [fac30/api_project--emma-kieran#9]
- Choose colour scheme and think about layout
- Redo basic front end so user input and return from openai looks nicer
- - Possibly login button so users dont have to access localhost3000/login for spotify api to be accessible

**Comment by electricblues** [comment]
by the way, we don't have to create a brand new project. We can just put the index.html, css & js files inside of the public directory, and have it so that whenever the user accesses "/" it just gives them the app.

Then, from the frontend code, we can directly call the api, so the user never navigates to any routes themselves

**Comment by giantflyingegg** [comment]
Yes thats the sort of thing i was aiming for, i just wasnt sure exactly how it would work
